samples/nrf9160/lwm2m_client/Kconfig

```diff
@@ -166,6 +166,13 @@ config APP_MCUBOOT_FLASH_BUF_SZ
 config APP_CUSTOM_VERSION
 	string "Application version. Leave empty to use NCS version."
 
+config APP_NEIGHBOUR_CELL_SCAN_INTERVAL
+	int "Interval time [s] between neighbour cell scans"
+	depends on LWM2M_CLIENT_UTILS_SIGNAL_MEAS_INFO_OBJ_SUPPORT
+	default 60
```

Comment:
Probably nitpicking, but should the default be a much more longer? Closer to 10 minutes?